index.html

```diff
@@ -332,10 +362,10 @@ <h3>
             </svg>
 
 
-            <textarea id="subject" name="subject" required>Enter Text Here</textarea>
-            <button type="submit">Submit</button><br>
+            <textarea id="subject" name="subject" required>Hello Marianne! Good to see you!</textarea>
+            <button type="submit">Get in touch</button><br>
           </form>
```

Comment:
- [ ] Kindly make sure the `Get in touch`  button follows the styling like on the Figma design(e.g.: background color, padding, text color,...) and it is also supposed to have some interactions styling such as: `hover`, `pressed`, `Enabled`,... (I suggest you also apply these changes on other buttons like(See Project, Get My Resume) as required for the project)
- [ ] You can get all details on this image from Figma: 
 
![btn_colors](https://user-images.githubusercontent.com/83042742/179358638-38c1b3e6-7045-4872-a0f5-b29cf4333232.png)